### ISSUE TITLE
[BUG] XCH balance show 0 in the reinstalation

### ISSUE THREAD

**Issue by Gaburieru-K7** [Chia-Network/chia-blockchain#9924]
### What happened?

I made a reinstallation of my SO. I reinstall the Chia. Use the 24 words. Wallet shows "fully synced" (the node and the wallet) but all my XCH still missing and now I have a new address.

???

Chia explorer shows me the coins in the first address, but in the reisntalation the address change, and not show anything in that new address the Chiaexplorer.

I can restore the first address in the Chia?

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by shaneo257** [comment]
is it your node or your wallet showing fully synced?

**Comment by Gaburieru-K7** [comment]
> is it your node or your wallet showing fully synced?

Both are in sync.

**Comment by esaung** [comment]
are you using the light wallet? and is that where you are seeing the missing balance? If so, please take a look at this article as the light wallet has implemented the hardened keys and as a result will show a different balance then what you see in your 1.2 full node wallet. 

https://www.chia.net/2022/01/10/understanding-the-changes-in-the-new-light-wallet.en.html

another option to look at is did you double check you used the correct 24 word recovery to see the correct wallet?

**Comment by Gaburieru-K7** [comment]
i

> are you using the light wallet? and is that where you are seeing the missing balance? If so, please take a look at this article as the light wallet has implemented the hardened keys and as a result will show a different balance then what you see in your 1.2 full node wallet.
> 
> https://www.chia.net/2022/01/10/understanding-the-changes-in-the-new-light-wallet.en.html
> 
> another option to look at is did you double check you used the correct 24 word recovery to see the correct wallet?

I have version 1.2.11, I reinstalled the S.O. I reinstalled the wallet, used all 24 keywords. The node and the wallet are in sync. My wallet address changed, the old balance disappeared. 

Is there a way to go back to the old address?

**Comment by Aleksandr898** [comment]
Помогите решить проблему!!!
Использую сейчас легкий кошелек , сдела перевод монет с биржи kukoin , монеты отправились но на кошельке их не видно вот TxHash 0x8511934f9298dc72e379067a3558ee4f2ffeb69e42b61a07c3d6e5f22af58638

**Comment by radchenk0** [comment]
Hello!

Surprisingly, I've just found myself in the exactly same situation.

My story. Probably it will help to point out what I'm doing wrong.

A couple of weeks ago, I decided to jump in the project. I created virtual machine(Debian) on my laptop(laptop is on Windows), installed light wallet in it, bought a few xch on centralized exchange and sent them to my receive address that I got once installed light wallet.
Then I double checked that I got them and have access to my wallet by removing virtual machine, creating a new one and restoring wallet from mnemonic. Wallet was correctly restored and I saw my balance that I'd expected.

A couple of days ago, I downloaded the latest full node installer(1.2.11) on my desktop(running on Windows), finished installation, restored the wallet from my mnemonic and started waiting when it finishes sync process. It took 4-5 days(working hours) and now I cannot see my xch.
Confirm that node and wallet are synced with blockchain
Also confirm that coins are not stolen via xchscan.com

Can anyone help or suggest something?
Kind regards

**Comment by radchenk0** [comment]
I was hoping that it's just gui related bug, but cli shows exactly the same information

What is going on, guys?

**Comment by radchenk0** [comment]
Okay, I've figured out. It looks like the full node app (12.11) is somehow behind the latest light wallet(1.3) in terms of development or releasing.
What I've done - I installed the latest light wallet, restored wallet from mnemonic, synced and ta-dam, it shows my balance correctly!
Seems like I'll be waiting for a new release of the full node app, I guess devs will fix this/provide the same feature from light wallet, so there will be no such issue/confusion in future
